### ISSUE TITLE
Error on facet load

### ISSUE THREAD

**Issue by johanvonboer** [humlab-sead/sead_query_api#25]
Following request results in an internal server error (500)
```javascript
fetch("https://seadserv.humlab.umu.se:8089/api/facets/load", {"credentials":"omit","headers":{"accept":"application/json, text/javascript, */*; q=0.01","content-type":"application/json; charset=UTF-8","sec-fetch-mode":"cors"},"referrer":"http://localhost:8080/","referrerPolicy":"no-referrer-when-downgrade","body":"{\"requestId\":1,\"requestType\":\"populate\",\"targetCode\":\"sites\",\"facetConfigs\":[{\"facetCode\":\"sites\",\"position\":1,\"picks\":[],\"textFilter\":\"\"}]}","method":"POST","mode":"cors"});
```

**Comment by roger-mahler** [comment]
```json
{
  
  "requestId": 1,
  "requestType": "populate",
  "targetCode": "sites",
  "facetConfigs": [
    {
      "facetCode": "sites",
      "position": 1,
      "picks": [],
      "textFilter": ""
    }
  ]
}
```

**Comment by roger-mahler** [comment]
Resolved by #27 and #28

**Comment by roger-mahler** [comment]
TriggerCode ought to always have a value, even when a single facet load is requested.
